controllers/helmrepository_controller.go

```diff
@@ -504,9 +514,11 @@ func (r *HelmRepositoryReconciler) garbageCollect(ctx context.Context, obj *sour
 	return nil
 }
 
-// eventLog records event and logs at the same time. This log is different from
-// the debug log in the event recorder in the sense that this is a simple log,
-// the event recorder debug log contains complete details about the event.
+// eventLogf records event and logs at the same time.
```

Comment:
I am not sure I fully understand the description. Does it "records events and logs at the same time" or does it "records an event and logs it at the same time"?